platforms/hyperledger-fabric/configuration/molecule/crypto/converge.yml

```diff
@@ -59,8 +62,17 @@
             grpc:
               port: 7054
         vault:
-          url: test_url
-        
+          url: http://localhost:9090/
```

Comment:
why repeat? this is already defined at top level

Reply:
Removed